From c732fc2c4ebd8062fbf54673fb36dbaffbf02981 Mon Sep 17 00:00:00 2001
From: Jianzheng Zhou <jianzheng.zhou@freescale.com>
Date: Thu, 15 Aug 2013 17:07:39 +0800
Subject: [PATCH 235/281] ENGR00275202 Mms: fix one NPE in Mms app

if not declare support mobile_mms networkAttributes but ethernet in config.xml
it will trigger this null pointer exception. Align this with sabresd.

Signed-off-by: Jianzheng Zhou <jianzheng.zhou@freescale.com>
---
 .../overlay/frameworks/base/core/res/res/values/config.xml        | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/sabreauto_6q/overlay/frameworks/base/core/res/res/values/config.xml b/sabreauto_6q/overlay/frameworks/base/core/res/res/values/config.xml
index 610a3f6..8bbb7ef 100755
--- a/sabreauto_6q/overlay/frameworks/base/core/res/res/values/config.xml
+++ b/sabreauto_6q/overlay/frameworks/base/core/res/res/values/config.xml
@@ -71,7 +71,15 @@
     <!-- the 6th element indicates boot-time dependency-met value. -->
     <string-array translatable="false" name="networkAttributes">
         <item>"wifi,1,1,1,-1,true"</item>
+        <item>"mobile,0,0,0,-1,true"</item>
+        <item>"mobile_mms,2,0,3,60000,true"</item>
+        <item>"mobile_supl,3,0,3,60000,true"</item>
+        <item>"mobile_hipri,5,0,4,60000,true"</item>
+        <item>"mobile_fota,10,0,3,60000,true"</item>
+        <item>"mobile_ims,11,0,3,60000,true"</item>
+        <item>"mobile_cbs,12,0,3,60000,true"</item>
         <item>"wifi_p2p,13,1,0,-1,true"</item>
+        <item>"bluetooth,7,7,2,-1,true"</item>
         <item>"ethernet,9,9,2,-1,true"</item>
     </string-array>
 
-- 
1.8.0

